tests/functional/spec/features/routing/number.spec.js

```diff
@@ -142,35 +130,9 @@ describe("Feature: Routing on a Number", () => {
   });
 
   describe("Less Than or Equal", () => {
-    describe("Given I start number routing less then or equal survey", () => {
-      before(() => {
-        browser.openQuestionnaire("test_routing_number_less_than_or_equal.json");
-      });
-
-      it("When I enter a less number, Then I should be routed to the correct page", () => {
-        $(NumberQuestionPage.answer()).setValue(23);
-        $(NumberQuestionPage.submit()).click();
-        expect(browser.getUrl()).to.contain(CorrectAnswerPage.pageName);
-      });
-
-      it("When I enter an equal number, Then I should be routed to the correct page", () => {
-        $(CorrectAnswerPage.previous()).click();
-        $(NumberQuestionPage.answer()).setValue(123);
-        $(NumberQuestionPage.submit()).click();
-        expect(browser.getUrl()).to.contain(CorrectAnswerPage.pageName);
-      });
-
-      it("When I enter a greater number, Then I should be routed to the incorrect page", () => {
-        $(CorrectAnswerPage.previous()).click();
-        $(NumberQuestionPage.answer()).setValue(546);
-        $(NumberQuestionPage.submit()).click();
-        expect(browser.getUrl()).to.contain(IncorrectAnswerPage.pageName);
-      });
-    });
-
     describe("Given I have number routing with a less than or equal to single condition", () => {
       before(() => {
-        browser.openQuestionnaire("test_routing_number_less_than_or_equal_single_condition.json");
+        browser.openQuestionnaire("test_new_routing_number_less_than_or_equal.json");
       });
```

Comment:
I think we should remove the single_condition references.

Reply:
I think Ive removed them all now, I cant find any more